Sortable.js

```diff
@@ -513,8 +514,8 @@
 				var	options = this.options,
 					fallbackTolerance = options.fallbackTolerance,
 					touch = evt.touches ? evt.touches[0] : evt,
-					dx = touch.clientX - tapEvt.clientX,
-					dy = touch.clientY - tapEvt.clientY,
+					dx = (touch.clientX - tapEvt.clientX) + this.options.fallbackOffset.x,
```

Comment:
add: `fallbackOffset = options.fallbackOffset,`
and: `this.options.fallbackOffset` -> `fallbackOffset`

👍